frontend/public/components/utils/units.js

```diff
@@ -34,10 +34,11 @@ const TYPES = {
     space: false,
     divisor: 1000,
   },
-  decimalBytesPerSec: {
-    units: ['Bps', 'KBps', 'MBps', 'GBps', 'TBps', 'PBps', 'EBps'],
+  decimalBitsPerSec: {
+    units: ['bps', 'Kbps', 'Mbps', 'Gbps', 'Tbps', 'Pbps', 'Ebps'],
```

Comment:
We should add some unit tests for this to frontend/__tests__/units.js

Reply:
Tests added. I noticed that `frontend/__tests__/units.js` is not run with `test-frontend.sh` as jest textRegex `"/__tests__/.*\\.spec\\.(ts|tsx|js|jsx)$"` does not match `units.js`. Is that on purpose ? Also quite a few `units.js` test are failing (even before this change).

Reply:
No, it's not on purpose :(

It looks like a result of https://github.com/openshift/console/commit/b4bc8c9d8b802fd1fb8dfa02a31a9adccb7998cb

cc @vojtechszocs

Reply:
I opened https://jira.coreos.com/browse/CONSOLE-1658

Reply:
#2204 reverts to the original `textRegex` pattern.

However, there's currently 8 files without `spec` in their name, but being actual tests:

```
./__tests__/components/utils/datetime.ts
./__tests__/public/co-fetch.js
./__tests__/resource.js
./__tests__/selector-requirement.js
./__tests__/selector.js
./__tests__/container.ts
./__tests__/probe.ts
./__tests__/units.js
```

Files under `__tests__` without `spec` in their name should only be helpers (utils, mocks, etc), otherwise the `spec` naming convention isn't that useful.